match2/wikis/rocketleague/match_group_legacy_default.lua

```diff
@@ -25,11 +29,18 @@ function p.get(templateid, bracketType)
 		winner = "map$1$win"
 	}
 
+	for n=1,lastround do
+		bracketData["R" .. n .. "M1header"] = "R" .. n
+		if LowerHeader[n] then
+			bracketData["R" .. n .. "M" .. LowerHeader[n] .. "header"] = "L" .. n
+		end
+	end
+
 	return bracketData
 end
 
 local lastRound
-function p._getMatchMapping(match, bracketData, bracketType)
+function p._getMatchMapping(match, bracketData, bracketType, LowerHeader)
 	local id = String.split(match.match2id, "_")[2] or match.match2id
 	id = id:gsub("0*([1-9])", "%1"):gsub("%-", "")
```

Comment:
Could use a comment to explain the regex, to save some mental energy for the reader